decidim-core/app/controllers/decidim/newsletters_controller.rb

```diff
@@ -12,8 +12,13 @@ class NewslettersController < Decidim::ApplicationController
     def show
       @organization = current_organization
       @newsletter = collection.find(params[:id])
-
-      @body = parse_interpolations(@newsletter.id, @newsletter.body[I18n.locale.to_s], current_user)
+      @user = current_user
```

Comment:
You should be able to use `current_user` in the views directly instead of the instance variable.